src/Integration/Laravel/Cache/composer.json

```diff
@@ -13,8 +13,8 @@
         "php": "^7.2.5 || ^8.0",
         "ext-json": "*",
         "async-aws/dynamo-db": "^1.1",
-        "illuminate/cache": "^6.18.13 || ^7.10 || ^8.0",
-        "illuminate/support": "^6.18.13 || ^7.10 || ^8.0"
+        "illuminate/cache": "^6.18.13 || ^7.10 || ^8.0 || ^9.0",
```

Comment:
The CI ran with the version 8 (see https://github.com/async-aws/aws/runs/4914754787?check_suite_focus=true#step:10:172)

Any chance to make the CI testing this new version?

Reply:
@jderusse you'll need the following in your composer.json:

```
    "minimum-stability": "dev",
    "prefer-stable": true
```

Reply:
We have a test with dev flag.

main composer.json file need to be updated too

https://github.com/async-aws/aws/blob/4aafa6bf7f3514c1cff14c94cd3f58cfb43401f3/composer.json#L24